Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -679,37 +762,50 @@ extension ExitTest {
         var backChannelWriteEnd: FileHandle!
         try FileHandle.makePipe(readEnd: &backChannelReadEnd, writeEnd: &backChannelWriteEnd)
 
-        // Let the child process know how to find the back channel by setting a
-        // known environment variable to the corresponding file descriptor
-        // (HANDLE on Windows.)
-        var backChannelEnvironmentVariable: String?
-#if SWT_TARGET_OS_APPLE || os(Linux) || os(FreeBSD) || os(OpenBSD)
-        backChannelEnvironmentVariable = backChannelWriteEnd.withUnsafePOSIXFileDescriptor { fd in
-          fd.map(String.init(describing:))
-        }
-#elseif os(Windows)
-        backChannelEnvironmentVariable = backChannelWriteEnd.withUnsafeWindowsHANDLE { handle in
-          handle.flatMap { String(describing: UInt(bitPattern: $0)) }
-        }
-#else
-#warning("Platform-specific implementation missing: back-channel pipe unavailable")
-#endif
-        if let backChannelEnvironmentVariable {
+        // Create another pipe to send captured values (and possibly other state
+        // in the future) to the child process.
+        var capturedValuesReadEnd: FileHandle!
+        var capturedValuesWriteEnd: FileHandle!
+        try FileHandle.makePipe(readEnd: &capturedValuesReadEnd, writeEnd: &capturedValuesWriteEnd)
+
+        // Let the child process know how to find the back channel and
+        // captured values channel by setting a known environment variable to
+        // the corresponding file descriptor (HANDLE on Windows) for each.
+        if let backChannelEnvironmentVariable = _makeEnvironmentVariable(for: backChannelWriteEnd) {
           childEnvironment["SWT_EXPERIMENTAL_BACKCHANNEL"] = backChannelEnvironmentVariable
         }
+        if let capturedValuesEnvironmentVariable = _makeEnvironmentVariable(for: capturedValuesReadEnd) {
+          childEnvironment["SWT_EXPERIMENTAL_CAPTURED_VALUES"] = capturedValuesEnvironmentVariable
```

Comment:
I'd find it nice if the two env vars used a similar naming convention as each other

Reply:
For the moment, their names represent what they are and what they do. In the future, if either/both become host to more functionality, we can rename them. There's no ABI stability concern here as both endpoints are the same version of Swift Testing.

Reply:
Understood that we're free to evolve them at any time and that's great. Still, I find them more self-describing if they conveyed their direction in the names. For example I don't know whether the "back channel" is going from parent to exit test or vice-versa without looking it up. Both pipes could be considered "back channels".

Can we incorporate this feedback if/when we elevate the feature to public, since that will involve removing "experimental" from the names?

Reply:
Let's save this discussion for a separate PR.